test/OrigamiMembershipToken.t.sol

```diff
@@ -596,6 +596,56 @@ contract MembershipTokenVotingPowerTest is OMTHelper {
         assertEq(token.delegates(mintee), delegatee);
     }
 
+    function testTransferMTTransfersVotingPower() public {
```

Comment:
```suggestion
    function testTransferMovesVotingPower() public {
```

Reply:
What about `testTransferOMTMovesVotingPower` <-- trying to call out the transfer of the actual Membership Token just to be clear. Maybe not necessary?

Reply:
It's in a test contract named `MembershipTokenVotingPowerTest`, but I leave it up to you :) This could be useful for focusing a single test if you had another test that had the same name in the governance token tests.